skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +325,67 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
+    def test_ancom(self):
```

Comment:
:+1:, this test method needs to be split into several smaller test methods that each test (ideally) one thing. Please give each test a meaningful name so its purpose is clear.